vmm/src/vm.rs

```diff
@@ -227,8 +228,10 @@ impl Vcpu {
     /// * `kernel_start_addr` - Offset from `guest_mem` at which the kernel starts.
     /// * `vm` - The virtual machine this vcpu will get attached to.
     pub fn configure(&mut self, kernel_start_addr: GuestAddress, vm: &Vm) -> Result<()> {
+        let mut cpuid = vm.cpuid.borrow_mut();
```

Comment:
I don't see why we're bothering with this complexity, we could simply `clone()` the `Cpuid` from here since we don't want those changes to affect the generic `Cpuid` provided by the `Vm`.

Reply:
My initial purpose of using RefCell is for performance consideration. Since normally the length of the cpuid entry array will be around 100 and each entry is 32 bytes so the clone() is about 3200 bytes copy.  But your are right, changing the cpuid here has side effect of changing the Vm level copy. In my latest version I used the clone() for simplicity, it's at least better than crosvm which calls KVM_GET_SUPPORTED_CPUID to obtain a new copy from KVM.